operations/index.md

```diff
@@ -333,6 +333,7 @@ Mechanisms used by the system to ensure the security and authentication of inter
        * [Change the LDAP Server IP Address for New LDAP Server Content](security_and_authentication/Change_the_LDAP_Server_IP_Address_for_New_LDAP_Server_Content.md)
        * [Remove the LDAP User Federation from Keycloak](security_and_authentication/Remove_the_LDAP_User_Federation_from_Keycloak.md)
        * [Add LDAP User Federation](security_and_authentication/Add_LDAP_User_Federation.md)
+       * [Keycloak User Management with Kcadm](security_and_authentication/Keycloak_User_Management_with_Kcadm.md)
```

Comment:
The title here should correspond with the heading in the file:

Keycloak User Management with kcadm.sh